controllers/hcpvaultsecretsapp_controller.go

```diff
@@ -128,7 +128,7 @@ func (r *HCPVaultSecretsAppReconciler) Reconcile(ctx context.Context, req ctrl.R
 		d, err := parseDurationString(o.Spec.RefreshAfter, ".spec.refreshAfter", r.MinRefreshAfter)
 		if err != nil {
 			logger.Error(err, "Field validation failed")
-			r.Recorder.Eventf(o, corev1.EventTypeWarning, consts.ReasonVaultStaticSecret,
+			r.Recorder.Eventf(o, corev1.EventTypeWarning, consts.ReasonHVSStaticSecret,
```

Comment:
`refreshAfter` is also used with rotating secrets (since they're fetched along with static secrets), so perhaps a more generic reason would be fitting here? `ReasonHVSAppConfig` or something similar?